sdk/framework/backend.go

```diff
@@ -528,9 +528,20 @@ func (b *Backend) handleRootHelp(req *logical.Request) (*logical.Response, error
 	// names in the OAS document.
 	requestResponsePrefix := req.GetString("requestResponsePrefix")
 
+	// Generic paths will primarily be used for code generation purposes.
+	// This will result in a dynamic mount path being placed instead of
+	// a hardcoded default path. For example /auth/approle/login would be replaced
+	// with /auth/{mountPath}/login. This will be replaced for all secrets
+	// engines and auth methods that are enabled.
+	genericMountPaths, ok := req.Get("genericMountPaths").(bool)
+
+	if !ok {
+		genericMountPaths = false
+	}
```

Comment:
If the typecasting works, I think this part is redundant, since false is the default value
```suggestion
```

Reply:
@averche If you omit an `ok` check wouldn't this panic if the casting fails?

Reply:
Yes, so this will fail without it when calling through the help endpoint, i.e. if it doesn't passthrough the `pathInternalOpenApi` call in logical_system.go. E.g. you're calling `http://127.0.0.1:8200/v1/auth/approle\?help\=1`

That gets handled differently b/c it's getting the mount path from the FieldData which is treating it like a bool, and there is a helper that will return the default value as specified in the FieldData for this. Thus why we need a check here and not in pathInternalOpenApi get call.